README.md

```diff
@@ -126,6 +126,44 @@ If using __o-table__ in silent mode, use the mixin `oTableBase' in your table st
 }
 ```
 
+#### Themes
+
+Themes allow Bower and Build Service users to request `o-table` with a different look and feel. Default themes include:
+
+1. primary - The primary FT brand, which includes uses of the FT paper colour.
+2. internal - A more neutral theme useful for internal products, tooling, or tech documentation.
+
+Using themes is not required and mixins can continue to be used without setting a theme, however doing so is a quick way to customise the look and feel of `o-table`. For base styles, which outputs minimal css, leave the theme unset.
+
+##### Origami Build Service Users
+
+To set the theme modify your Origami Build Service request to include your desired theme e.g. `o-table@^6.2.1:primary`. If the theme is not set or found only base styles will be returned.
```

Comment:
s/primary/masterbrand

We talked about requesting themes like this vs a single parameter passed in to the build service